includes/abstracts/abstract-batch.php

```diff
@@ -329,7 +349,7 @@ public function process_results( $results ) {
 		 */
 		$failed_status = apply_filters( 'loco_batch_failed_status', 'failed' );
 
-		foreach ( $results as $result ) {
+		foreach ( $results as $i => $result ) {
```

Comment:
We should try to avoid single character variables like `$i` in favor of something more descriptive (like `$key`, or if it's the result ID maybe `$result_id`).